packages/schemas/core/file.py

```
@@ -0,0 +1,143 @@
+    extractor_version: str = Field(
+        ...,
+        description="Version of the extractor that created this entity",
+        examples=["1.0.0", "1.2.0"],
+    )
+
+    @field_validator("extraction_tier")
+    @classmethod
+    def validate_extraction_tier(cls, v: str) -> str:
+        """Validate extraction_tier is A, B, or C."""
+        if v not in ("A", "B", "C"):
+            raise ValueError("extraction_tier must be A, B, or C")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)